File: TersKosegen.py
def anti_diagonal(sayi):
    a = sayi-1
    for k in range(sayi):
        satir = []
        for i in range(sayi-1):
            satir.append(0)
        satir.insert(a, 1)
        a -= 1
        print(satir)

sayi = int(input("Bir sayı giriniz: "))
anti_diagonal(sayi)

File: TersKosegen.swift
//  Created by Kağan on 15.01.2017.
//  Copyright © 2017 Kağan Utku Kılıçlı. All rights reserved.

import Foundation

print("bir sayi giriniz:")
if var sayi = Int(readLine(strippingNewline: true)!) {
    for i in 1...sayi {
        for j in 1...sayi {
            print( i+j == sayi+1 ? "1" : "0" , terminator:"\t")
        }
        print("",terminator:"\n\n")
    }
}


